package.json

```diff
@@ -38,6 +38,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "ajv": "^6.12.3"
+    "ajv": "^6.12.3",
```

Comment:
Chai is a devDependency since it's not needed in the regular runtime/consumption of the library.